configs/config.dev.yaml

```diff
@@ -194,6 +194,10 @@ oncall-key: abc # API key for oncall auth
 applications:
  - dummy_app
 
+enable_triage_context: False
```

Comment:
rename config to something more descriptive of what it does (like enable_default_metavariables_in_context) and add a comment explaining config option

Reply:
done